### PR TITLE
optimizer-addition

### PR DESCRIPTION
**Pull request by viridaria**

### CODE REVIEW

**Comment by catlook** [comment]
@viridaria review my edits and see if you agree :).

File: docs/rackspace-elastic-engineering-and-optimizer/index.rst [hunk 1]
```diff
@@ -5,14 +5,14 @@ Rackspace Elastic Engineering and Optimizer +
...
-for your needs at any stage of your cloud lifecycle. Our services provide you
-access to certified cloud specialists through a flexible POD-based model,
-experts who can implement industry best cloud monitoring technologies, cost
-optimization recommendations, experts to solve your IT challenges, and other
+their cloud journey. That’s why we provide a set of support offers that address
+your needs at any stage of your cloud lifecycle. Our services provide you
+access to certified cloud specialists through a flexible POD-based model. We have
+experts who can implement industry-best cloud monitoring technologies, suggest cost
+optimization recommendations, solve your IT challenges, and offer other
 value-added services, all backed by AWS certified engineers and architects.
```
Comment (ltvolks) [inline docs/rackspace-elastic-engineering-and-optimizer/index.rst:13]: Should final clause be separated? Also, we should probably review for anything that mentions AWS in these  GCP docs 😄 

```suggestion
value-added services — all backed by GCP-certified engineers and architects.
```

File: docs/rackspace-elastic-engineering-and-optimizer/index.rst [hunk 2]
```diff
@@ -42,47 +42,48 @@ between Rackspace Technology and the customer.
...
-   * - Advice and Support
-     - * Answer questions related to cloud usage, best practices, and make recommendations
+   * - Advice and support
+     - * Answer questions and make recommendations related to cloud usage and best practices
+
 
 .. _monitoring_resolution:
 
-Advanced Monitoring and Resolution
+Advanced monitoring and resolution
```
Comment (ltvolks) [inline docs/rackspace-elastic-engineering-and-optimizer/index.rst:78]: Is this a product/service offering name, and should it retain capitalization?

Comment (viridaria) [reply docs/rackspace-elastic-engineering-and-optimizer/index.rst:78]: it is a product

**Pull request merged**